xml/System.Security.Cryptography/CryptographicOperations.xml

```diff
@@ -89,9 +89,19 @@ Fixed-time behavior is guaranteed in all other cases, including when `left` and
         <Parameter Name="buffer" Type="System.Span&lt;System.Byte&gt;" />
       </Parameters>
       <Docs>
-        <param name="buffer">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="buffer">The buffer to fill with zeros.</param>
+        <summary>Fills the provided buffer with zeros.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This method exists to future-proof against potential optimizations in the .NET runtime which could eliminate memory writes that are provably not followed by memory reads,
+similar to optimizations made by C and C++ compilers.
+No such optimizations are planned at this time.
```

Comment:
I think this sentence could confuse users. What does "at this time" mean? They would not know how long ago this text was added, or if it's out of date. Maybe we can reword this to be more specific?
```suggestion
No such optimizations are planned at this time.
```

Reply:
Sometimes we do say things as "as of late 2019."

e.g. https://github.com/dotnet/docs/search?q=%22as+of%22&unscoped_q=%22as+of%22
(conceptual docs have date published though so we can be a bit more vague.

Reply:
> Maybe we can reword this to be more specific?

It's an open-ended defense.  The point of the method is "maybe one day we'd do this, so you should declare your intent now and be safe in the indefinite-and-maybe-never-going-to-happen future".  I'm happy to accept suggestions, but leaving off the trailing sentence I feel implies we're planning them... but we're not, it's just theoretical.

Reply:
No, let's keep the trailing sentence. I can't think of a better way to reword this, and it's no big deal. I'll approve the PR, and if @mairaw has a suggestion to get it reworded, let's consider it.

Reply:
While I think this is a little different because it's talking about what's *not* planned for the future, there is [this guidance](https://review.docs.microsoft.com/en-us/help/contribute/contribute-get-started-channel-guidance?branch=master#content-that-does-not-belong-in-a-docsmicrosoftcom-article-or-repository) in our contributor's guide:

Future product plans: Do not publish statements about future product plans in technical documentation. Technical documentation should describe only what is possible in the released product. Why?

It’s not actionable: Technical content should be about what customers can and should do today, not what they might be able to do some day.
It erodes trust: If a writer says a feature is coming soon and it is delayed or canceled, the statement risks eroding trust for all our technical documentation.
Looks like marketing: Talking about the future looks more like marketing material (aspirational rather than reality).

Reply:
^ @bartonjs Genevieve found good guidance and she has a point, in case you may want to reconsider removing the sentence about future plans.